package.json

```diff
@@ -81,7 +81,8 @@
     "overrides": {
       "@nuxtjs/i18n": "link:.",
       "consola": "^3",
-      "unimport": "3.13.4"
+      "unimport": "3.13.4",
+      "rollup": "4.28.1"
```

Comment:
why do we need to override? 🤔

Reply:
This is to resolve the type mismatch for the nitro plugins like [`acfc08f` (#3273)](https://github.com/nuxt-modules/i18n/pull/3273/commits/acfc08f1296b5f55cc8424786160f1131c6f7f9f#diff-126f9d42f327404ca57cb5a71d7c101fe1b0ee34dc04757b61cf37d09b21f257L64), I was wondering how Nuxt isn't running into the same issue and found they also override the resolution for rollup https://github.com/nuxt/nuxt/blob/main/package.json#L59